java/src/jmri/implementation/DefaultConditional.java

```diff
@@ -68,7 +68,7 @@
 
     public static final boolean PARKS_DEBUG = false;
```

Comment:
What is this?

It seems that it would be better to simply use the log.debug() statements and set logging correctly instead of recompiling to enable debug statements.

I think it should be removed. (I only noticed because of changes to the logging below).

Reply:
I don't know.  It was added a long time ago, before July 2011 according to Git history.

Apparently someone named Parks wanted **println** statements in addition to log.debug.

I agree that it should be removed.  If you want, I can included that in my next set of Conditional tweaks.

Dave Sand